webviz_config/templates/webviz_template.py.jinja2

```diff
@@ -113,6 +113,21 @@ else:
             ])]),
         html.Div(className="pageContent", id="page-content")])
 
+plugins = []
+{% for page in pages %}
+{% for content in page.content %}
+{% if content is not string %}
+plugins.append(webviz_config.plugins.{{ content._call_signature[0] }})
```

Comment:
I think we want to create the `plugins` list prior to the for loop on line 73 (initializing plugins can be time consuming, depending on plugin logic, and it would be nice if line 124 and 79 could be using the same initialized plugins).

@asnyv can probably help you out if you want more details.